oauth2client/client.py

```diff
@@ -1222,6 +1227,31 @@ def create_scoped(self, scopes):
         """
         return self
 
+    @classmethod
+    def from_json(cls, s):
+        from oauth2client.service_account import _ServiceAccountCredentials
+        s = _from_bytes(s)
+        data = json.loads(s)
+
+        # A GoogleCredentials.from_stream return value might be a
```

Comment:
I think a more appropriate comment would be

```python
# We handle service_account._ServiceAccountCredentials since it is a
# possible return type of GoogleCredentials.get_application_default()
```